jobs/integr8ly/codeready-test.yaml

```diff
@@ -0,0 +1,84 @@
+---
+
+- job:
+    name: codeready-tests
+    project-type: pipeline
+    sandbox: true
+    parameters:
+      - string:
+          name: REPOSITORY
+          default: 'https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git'
+          description: 'QE Git repository URL.'
+      - string:
+          name: BRANCH
+          default: 'master'
+          description: 'Branch of the repository'
+      - string:
+          name: CLUSTER_URL
+          description: 'URL of cluster on which the tests will be executed.'
+      - string:
+          name: NAMESPACE_PREFIX
+          description: "Value used to prefix the names of the namespaces created during Integr8ly installation"
+      - string: 
+          name: ADMIN_USERNAME
+          default: 'admin@example.com'
+          description: 'User name to login to Integreatly cluster.'
+      - string: 
+          name: ADMIN_PASSWORD
+          default: 'Password1'
+          description: 'Password to login to Integreatly cluster.'
+      - string:
+          name: EVALS_USERNAME
+          default: 'evals11@example.com'
+          description: 'Evals account email address used to access the available consoles. This test expects that this user started the Walkthrough 1A!!'
```

Comment:
Is it really true that the user needs to start wt1A for this test?

Reply:
No :)